lldb/source/Plugins/Process/Linux/NativeProcessLinux.cpp

```diff
@@ -444,6 +461,88 @@ NativeProcessLinux::NativeProcessLinux(::pid_t pid, int terminal_fd,
   SetState(StateType::eStateStopped, false);
 }
 
+llvm::Expected<std::vector<::pid_t>> NativeProcessLinux::Seize(::pid_t pid) {
+  Log *log = GetLog(POSIXLog::Process);
+
+  uint64_t options = GetDefaultPtraceOpts();
+  Status status;
+  // Use a map to keep track of the threads which we have attached/need to
+  // attach.
```

Comment:
seized / need to seize

Reply:
In fact why does this say attach a lot. Are we attaching **by** seizing, or are seize and attach two different things completely?